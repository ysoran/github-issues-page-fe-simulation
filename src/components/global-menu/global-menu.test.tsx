import React from 'react';
import { shallow } from 'enzyme';
import GlobalMenu from './global-menu';

const app = () => {
    const wrapper = shallow(<GlobalMenu />);
    return { wrapper };
};

describe('Global Menu', () => {
    it('should render snapshot', () => {
        const { wrapper } = app();
        expect(wrapper).toMatchSnapshot();
    });
});
