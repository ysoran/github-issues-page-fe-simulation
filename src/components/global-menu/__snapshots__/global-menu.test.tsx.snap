// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global Menu should render snapshot 1`] = `ShallowWrapper {}`;
